README.md

```diff
@@ -25,13 +25,22 @@ It allows users to quickly spin up custom testnets and dev environments to test
 
 ### Maintained Branches
 
+#### Backport Policy:
+Strangelove maintains `n - 1` branches of interchaintest, `n` being current `main`.
```

Comment:
Technically, it's `n` and `n-1`.

IOW, we support the current version and previous version, nothing older.